src/views/Package/Package.tsx

```diff
@@ -19,16 +20,20 @@ interface PathParams {
 
 export const Package: FunctionComponent = () => {
   const { name, scope, version }: PathParams = useParams();
-  const [requestAssembly, assemblyResponse] = useRequest(createAssembly);
+  const [requestMarkdown, markdownResponse] = useRequest(fetchMarkdown);
+  const [requestAssembly, assemblyResponse] = useRequest(fetchAssembly);
   const [requestMetadata, metadataResponse] = useRequest(fetchMetadata);
+
   const q = useQueryParams();
   const [language] = useLanguage();
+  const submodule = q.get(QUERY_PARAMS.SUBMODULE) ?? "";
 
   useEffect(() => {
     void requestMetadata(name, version, scope);
+    void requestMarkdown(name, version, language, scope, submodule);
     void requestAssembly(name, version, scope);
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [name, scope, version]);
+  }, [name, scope, version, language]);
```

Comment:
Follow-up from standup:
> Changing Submodule does not refetch the data

The solution we should implement here would be to add a separate `useEffect` specifically for the markdown which will subscribe to changes to the `language` and `submodule`

```tsx
useEffect(() => {
  void requestMarkdown(name, version, language, scope, submodule);
}, [name, scope, version, language, submodule]);
```

The reason we would want this to be a separate effect is b/c we do not need to request the assembly or metadata whenever language or submodules change since assembly & metadata information will be the same regardless

Reply:
Cool thanks. Done 👍